BigQuery/src/BigQueryClient.php

```diff
@@ -807,4 +807,91 @@ public function getServiceAccount(array $options = [])
         $resp = $this->connection->getServiceAccount($options + ['projectId' => $this->projectId]);
         return $resp['email'];
     }
+
+    /**
+     * Returns a copy job configuration to be passed to either
+     * {@see Google\Cloud\BigQuery\BigQueryClient::runJob()} or
+     * {@see Google\Cloud\BigQuery\BigQueryClient::startJob()}. A
+     * configuration can be built using fluent setters or by providing a full
+     * set of options at once.
+     *
+     * Example:
+     * ```
+     * $copyJobConfig = $bigQuery->copy()->sourceTable($otherTable)->destinationTable($myTable);
```

Comment:
Easier to read when this is spread across multiple lines:

```suggestion
     * $copyJobConfig = $bigQuery->copy()
     *     ->sourceTable($otherTable)
     *     ->destinationTable($myTable);
```